readthedocs/doc_builder/backends/mkdocs.py

```diff
@@ -39,17 +39,8 @@ class BaseMkdocs(BaseBuilder):
 
     """Mkdocs builder."""
 
-    use_theme = True
-
-    # The default theme for mkdocs (outside of RTD) is the 'mkdocs' theme
-    # For RTD, our default is the 'readthedocs' theme
-    READTHEDOCS_THEME_NAME = 'readthedocs'
-
-    # Overrides for the 'readthedocs' theme that include
-    # search utilities and version selector
-    READTHEDOCS_TEMPLATE_OVERRIDE_DIR = (
-        '%s/readthedocs/templates/mkdocs/readthedocs' % settings.SITE_ROOT
-    )
+    # The default theme for mkdocs is the 'mkdocs' theme
+    DEFAULT_THEME_NAME = 'mkdocs'
```

Comment:
Why not default to the rtd theme when users don't have a `theme` or `theme.name`?

Reply:
Starting with mkdocs>=1.0, theme is required. Also, this is another source of confusion where Read the Docs does something different than the default MkDocs functionality. I believe we shouldn't be changing users' intents.

Reply:
that makes sense, I wonder if this is good for old projects, all of them were building with the rtd theme, right?

Reply:
I like to keep the same behavior outside than inside RTD.

I want to echo Santos' question: what would happen with old projects that weren't specifying the them? Will they start building with the `mkdocs` theme? If so, I suppose we should do something to avoid that

Reply:
Assuming they target `mkdocs<1.0` and don't specify a theme, they will start to build with the `mkdocs` theme. This would be the same behavior as if they built locally. Starting with `mkdocs>=1.0`, they will fail to build with an error because theme is required.

Frankly, I think switching them to the `mkdocs` theme is the correct behavior as then their project builds with the same theme inside and outside RTD. In general, however, I suspect not a lot of projects do not specify a theme.

Reply:
I put some thoughts [here](https://github.com/rtfd/readthedocs.org/pull/4556#issuecomment-416653266) about this. I moved the discussion a bit since it references multiple comments.